test/test_problems.py

```diff
@@ -7,9 +7,12 @@
 import tools
 import problem
 import config
+import util
 
 # Run `bt run` on these problems.
-PROBLEMS = ['hello', 'helloproblemtools', 'different', 'fltcmp', 'boolfind', 'guess']
+PROBLEMS = ['hello', 'helloproblemtools', 'different', 'fltcmp', 'boolfind', 'guess'] + [
+    'hellounix' if not util.is_mac() and not util.is_windows() else []
```

Comment:
Consider
```
…'fltcmp'] + (['hellounix', 'boolfind', 'guess'] if  not util.is_mac()  else [])
```

Reply:
`guess` i'll fix differently using the less strict checking for distinct error types. I'm still not quite sure why `boolfind` crashes though

Reply:
both boolfind (remove `-static`) and guess (more lenient check) should be fixed now. can you check?